react/getting_started_with_react/keys_in_react.md

```diff
@@ -0,0 +1,124 @@
+### Introduction
+
+In this lesson we will cover keys in React. Keys are special props for our components and we'll learn why they are used.
+
+### Lesson Overview
+
+By the end of the lesson you should be able to answer the following:
+
+- Why are keys and why does React need them. 
+- What are good and bad examples of keys. 
```

Comment:
```suggestion
- What are good and bad examples of keys?
```

also turned into a question. Alternatively it could be written as a sentence:
> What good and bad examples of keys are